rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -390,6 +418,31 @@ impl TunnelManager {
         babel.monitor(&tunnel.iface_name)?;
         Ok(())
     }
+
+    /// Given a LocalIdentity, connect to the neighbor over wireguard
```

Comment:
How's this function distinct from `open_tunnel`? It's got the same description and yet it exists

Reply:
This opens a tunnel, but _does not connect over to the neighbor, only opens a wg interface which listens_, sorry I'll update the docs